File: 6.Two Pointer/python_solution/15.VerifySubsequence.py

def verifySubsequence(s,t):
    while i < len(s) and j < len(t):
      if s[i] == s[j]:
        i+=1
      j+=1

    if(i == len(s)):
      print("true")
    else:
      print('false')
  




s = "abc"
t = "ahbgdc"
res = verifySubsequence(s,t)
print(res)

File: 6.Two Pointer/javascript_solution/15.VerifySubsequence.js
function verifySubsequence(s, t) {
    let i = 0
    let j = 0;

    
    while (i < s.length && j < t.length) {
        if (s[i] === t[j]) {
            i++;
        }
        j++;
    }

    if (i === s.length) {
        console.log("true");
    } else {
        console.log("false");
    }
}

let s = "abc";
let t = "ahbgdc";
verifySubsequence(s, t);
